### PR TITLE
Add a blade component for the front page blocks.

### PR DESCRIPTION
**Pull request by emil-nasso**
A blade component for the front page blocks minimizes the boilerplate
for adding new blocks. It also makes it easier for a service
to override the front view and add their own blocks.

### CODE REVIEW

**Comment by emil-nasso** [comment]
Question: Do we need to support php 7.4?

The tests are failing because I was using php 8 constructor property promotion.

I could easily change it...unless we want to support only php 8+. :)

**Comment by ttrig** [comment]
> Question: Do we need to support php 7.4?
> 
> The tests are failing because I was using php 8 constructor property promotion.
> 
> I could easily change it...unless we want to support only php 8+. :)

Done, https://github.com/glesys/butler-service/pull/64 :tada:

**Review by ttrig — APPROVED** [review]

**Review by wecc — APPROVED** [review]

File: resources/views/components/front-block.blade.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<div {{ $attributes->merge(['class' => 'bg-white rounded p-4 mb-4']) }}>
+  <div class="flex flex-row mb-2">
+    <div class="flex-1 font-bold tracking-wide uppercase">{{ $title }}</div>
+    @if ($url)
+        <div class="rounded bg-gray-100 border px-2">
```
Comment (wecc) [inline resources/views/components/front-block.blade.php:5]: Indentation varies between 2 and 4 spaces in this file.

Comment (emil-nasso) [reply resources/views/components/front-block.blade.php]: Fixed

**Pull request merged**